app_dart/lib/src/model/appengine/time_series.dart

```diff
@@ -111,6 +111,18 @@ class SerializableTimeSeries {
   @KeyConverter()
   Key get key => series.key;
 
+  Map<String, dynamic> get facade {
+    return <String, dynamic>{
+      'Archived': series.archived,
+      'Baseline': series.baseline,
+      'Goal': series.goal,
+      'ID': series.timeSeriesId,
+      'Label': series.label,
+      'TaskName': series.taskName,
+      'Unit': series.unit,
```

Comment:
nitty nit: I would add branch here and update the `TimeSeriesValue`.toJson() to not serialize it. Every single time series value shared will be repeating the same bytes. (I think it's fine to just add a TODO)

Reply:
I was thinking about that too. But it didn't work out based on current logic in https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/request_handlers/get_benchmarks.dart
It queries recent 50 commits, and recent 50 TimeseriesValue, then if the `sha` matches between those two entities, a point is recorded, otherwise it will be noted as missing value.
When we consider branching, different branches may share same `sha`. Results will be messed up.
Another reason why I add the branch property to TimeseriesValue: existing logic queries data based on timestamp, which is the latest 50 records. If without branch property, there is no way to query the exact records of one specific branch. For example: in the latest 50 commits, if 48 is with `master`, 2 with `release branch`, the dashboard will show only the 2 records for that release branch, even though we may have older records beyond the 50 limit.

Reply:
I agree with the model needing to have the branch field for TimeSeriesValue. Only the backend needs to know about what branch a TimeSeriesValue is on. The frontend shouldn't need to further parse the data.

This was moreso on just hiding the TimeSeriesValue.branch field when returning the JSON.

Reply:
`branch` in TimeseriesValue is only as a Property, rather than a JsonKey: https://github.com/flutter/cocoon/blob/9f90c73ab083b49c65c43cba871bd6f6652e74ad/app_dart/lib/src/model/appengine/time_series_value.dart#L69.
So it should not be parsed in the frontend side.